v1.7.0 release (#554)
nerves-project/nerves: CHANGELOG.md

# Changelog

## v1.7.0

Nerves 1.7.0 removes support for creating OTP releases using Distillery and

> only supports using Elixir releases which bumps the minimum supported version
> of Elixir to 1.9.

Reviewer: ```suggestion
only supports using Elixir releases. As a result, the minimum supported version
of Elixir is now version 1.9.
```
